ApollosProject/apollos-prototype
# Clear Algolia Index Before Doing a Full Index.

## DESCRIPTION

### What does this PR do, or why is it needed?

Clears the Algolia index before attempting a full index. We do this because, while additions are always caught by the full index, any content that gets deleted will not actually be removed from the index.

### What design trade-offs/decisions were made?

### How do I test this PR?

I suggest you test this once in master and once on this branch.

1. Create a new piece of content in ROCK and publish it
1. Uncomment ln 11 in `apollos-data-connector-algolia`
2. Start your server, waiting for the index to finish
3. Search for your content, you should find it
4. Delete your content
5. Run the index again, waiting for it to finish
6. Search for your content again; if you're on this branch, you should not find it

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
